change/office-ui-fabric-react-eda7deda-1aa0-4a9c-a18b-8b073d191baf.json

```diff
@@ -0,0 +1,10 @@
+{
+  "type": "patch",
+  "comment": {
```

Comment:
Not sure what's up with this... Can you try deleting it, run `yarn`, and then run `yarn change` again?

Reply:
oh that's weird. I definitely typed a comment in there 🤔